install/update.rst

```diff
@@ -6,10 +6,17 @@ Updating Zammad
 Source update
 =============
 
-1. Download Zammad to your system
+1. Ensure your System meets the requirenments
+---------------------------------------------
+
+Before updating your installation, please ensure that your system  meets Zammads current 
+:doc:`Software requirenments </prerequisites/software>`.
```

Comment:
This seems like a strange thing to add to update instructions. Can dependencies change across a minor version bump? I would consider that a **breaking change**, which would require a major version bump according to [semantic versioning](https://semver.org/).

In any case, here's a rewording. If the above is true, consier adding this sentence just before "Before proceeding...":

> Dependencies may change with each update.

```suggestion
1. Check your dependencies
--------------------------

Before proceeding, double-check that your system environment
matches :doc:`Zammad’s requirements </prerequisites/software>`.
```